django/http/request.py

```diff
@@ -51,6 +51,15 @@ class HttpRequest:
     _encoding = None
     _upload_handlers = []
 
+    non_picklable_attrs = frozenset(
+        [
+            "resolver_match",
+            "environ",
+            "META",
```

Comment:
I'd be good to identify what in `META` is non-pickleable and maybe remove only this value :thinking: